docs-article-templates/src/helper/github.ts

```diff
@@ -0,0 +1,41 @@
+"use-strict";
+
+import * as fs from "fs";
+import os = require("os");
+import * as path from "path";
+import * as vscode from "vscode";
+import * as common from "./common";
+
+// to-do: change the console output to another form of user notification prior to production release.
+/* tslint:disable:no-console */
```

Comment:
super minor, might consider in the future just oneline turnoffs so, if you're like me, you don't accidentally add console where you don't want them

Reply:
Good point